### PR TITLE
exercise 1 program (but without formal comments)

### PR DESCRIPTION
**Pull request by helena-rg**

### CODE REVIEW

**Comment by helena-rg** [comment]
Hi, sorry I don't know if there is any issue with the recent commits that I have done (for this new pull requests)

File: exercises/rh/ex1/e1.f90 [hunk 1]
```diff
@@ -1,25 +1,105 @@
...
   implicit none
 
-INTEGER :: i
-INTEGER :: n ! number of particles
-type(particle3d), dimension(:), allocatable :: particles ! particle type array (contains the particles info)
-real, dimension(:), allocatable :: m
+  INTEGER :: i, j ! loop indexes
+  INTEGER :: rc ! variable to read line of input file (with the i.c.)
+  INTEGER :: n = 0 ! number of particles
+  REAL :: dt, t_end, t, dt_out, t_out = real64 ! time variables
```
Comment (hpparvi) [inline exercises/rh/ex1/e1.f90]: Great work, but you might still want to take a look at how to set the kind of your real variables from the [lecture slides](https://github.com/hpparvi/ull_tp_2425/blob/main/lectures/l03/l03b.pdf).

**Comment by hpparvi** [comment]
Excellent work Helena! The code is looking good, but you may still want to revise the way you set the [real variable precision](https://github.com/hpparvi/ull_tp_2425/blob/main/lectures/l03/l03b.pdf).

**Comment by helena-rg** [comment]
Hi, I think I fixed the setting of double-precision reals, is it correct now? Another question, if I use kind to set the double precision, it is not necessary to use the module iso_fortran_env, is it?

**Comment by hpparvi** [comment]
Hi, using `kind=8` instead of `real64` imported from `iso_fortran_env` is one possibility, but I would recommend using `real64`. The reason for this is that the meaning of the kind value is compiler-dependent, so a different compiler could, in theory, interpret `kind=8` as something other than a double-precision float.

Otherwise everything is looking very good!

**Comment by helena-rg** [comment]
Hi, I think I change to real64 all of the real variable. The program seems to work correctly now, it hasn't got anything else strange, has it?
If I have time I will try to include the description in the readme file. Thanks

**Comment by hpparvi** [comment]
Looks good now, great work!

**Pull request merged**